pytest_flake8.py

```diff
@@ -213,21 +213,12 @@ def check_file(path, flake8ignore, maxlength, maxdoclenght, maxcomplexity,
     if statistics:
         args += ['--statistics']
     app = application.Application()
-    prelim_opts, remaining_args = app.parse_preliminary_options(args)
-    config_finder = config.ConfigFileFinder(
-        app.program,
-        prelim_opts.append_config,
-        config_file=prelim_opts.config,
-        ignore_config_files=prelim_opts.isolated,
-    )
-    app.find_plugins(config_finder)
-    app.register_plugin_options()
-    app.parse_configuration_and_cli(config_finder, remaining_args)
+    app.plugins, app.options = parse_args(args)
     if flake8ignore:
         app.options.ignore = flake8ignore
     app.make_formatter()  # fix this
     app.make_guide()
-    app.make_file_checker_manager()
+    app.make_file_checker_manager([])
     app.run_checks([str(path)])
```

Comment:
Hello, correct me if I'm wrong because I'm not an expert. I am wondering how you know that `run_checks()` needs that one argument (`[str(path)]`), because in [run_checks()](https://github.com/PyCQA/flake8/blob/d016204366a22d382b5b56dc14b6cbff28ce929e/src/flake8/main/application.py#L92) is not other argument than `self`.

Reply:
Oops I might have a wrong setup will retry